api/api.py

```diff
@@ -137,6 +137,43 @@ def k8s_resource(name: str, yaml: Union[str, Blob] = "", image: Union[str, FastB
   """
   pass
 
+def filter_yaml(yaml: Union[str, List[str], LocalPath, Blob], labels={}, name='', kind='') -> Tuple[Blob, Blob]:
```

Comment:
you could add type annotations to `labels`, `name`, and `kind` instead of putting them in manually down below, to get consistent formatting